docs/02-Scales.md

```diff
@@ -231,7 +231,7 @@ minUnit | String | 'millisecond' | The minimum display format to be used for a t
 
 When providing data for the time scale, Chart.js supports all of the formats that Moment.js accepts. See [Moment.js docs](http://momentjs.com/docs/#/parsing/) for details.
 
-#### Display Formats
+### Display Formats
```

Comment:
Not sure that's the way to fix it because "Display Format" belongs to "Time Scale", which is at level 3. Actually all other sub-sections are also broken. For 2.5, we are planing to move the documentation to GitBook, so hopefully it should fix all the doc anchors.